### PR TITLE
Issue#325

### PR DESCRIPTION
**Pull request by ItayNaumann**

### CODE REVIEW

File: lib/views/pc/status/fetch_status.dart [hunk 1]
```diff
@@ -110,7 +110,7 @@ Stream<List<StatusItem<MatchIdentifier, Match>>> fetchStatus(
           scouter: e["scouter_name"] as String,
         );
       },
-      (final MatchIdentifier identifier, final List<Match> currentValues) {
+      (final MatchIdentifier identifier, final List<Match> scoutedTeams) {
```
Comment (giladwo) [inline lib/views/pc/status/fetch_status.dart:113]: How is `scoutedTeams` a list of `Match`es?

**Comment by ItayNaumann** [comment]
it has a conflict with what i change, please someone help @giladwo @itamarsch @amitcha06

**Comment by giladwo** [comment]
Rebase and squash on an updated `master`.
If this doesn't mean much to you, complete [the exercises](https://gitexercises.fracz.com/) first.

**Review by amitcha06 — COMMENTED** [review]
these commits are way too small, no need for them (the naming ones) to be separate, for example. same goal => same commit (ususally)

**Comment by giladwo** [comment]
> these commits are way too small, no need for them (the naming ones) to be separate, for example.

In this case the issue is that most of these commits do not belong here.

Commits are cheap, so there's nothing wrong with many small commits (as long as each of them builds successfully and preferably has a a distinctive feature).

**Pull request closed without merging**